core/src/queue.rs

```diff
@@ -18,7 +18,7 @@ use crate::prelude::*;
 
 impl AcceptedTransaction {
     // TODO: We should have another type of transaction like `CheckedTransaction` in the type system?
-    fn check_signature_condition(&self, wsv: &WorldStateView) -> Result<MustUse<bool>> {
```

Comment:
Wasn't there a difference between `Error` and `MustUse(false)`? Or do want to use previous `Error` as just `false`

Reply:
Well, I just wanted to process err in one place and as I noticed, this and derived methods have low chances to return a system error, but high chances to return business error which is SignatureCondition. So, I decided remove result from returning.